index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>Apple</title>
+  <link rel="stylesheet" href="main.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <!-- navbar starts -->
+    <div class="header">
+      <table border="1">
+        <tr>
+
+          <td style="border-top-left-radius: 6px; border-bottom-left-radius: 6px;"><button type="button" name="button">
+              <img src="./image/apple_logo.png" alt=""></button></td>
+          <td style="background-color: #222;"></td>
+          <td><button type="button" name="button"><img src="./image/store.png" alt=""></button></td>
```

Comment:
Not everything that is clicked must be a button. You can have a `div`, `span` or any other element look like a `button` and not have a `button` tag. As a rule of thumb, use `button` when you want to submit information. If it's just a link use a `div` or something else